src/main.rs

```diff
@@ -204,7 +204,7 @@ fn find_variables(val: &hcl::Value) -> Vec<String> {
             let tmpl = hcl::Template::from_str(&s).unwrap();
             let elems = tmpl.elements();
             for elem in elems {
-                println!("{:?}", elem);
+                // println!("{:?}", elem);
```

Comment:
remove?

Reply:
This is v helpful for debugging the parsed hcl values and the implementation is not complete, I'll leave a comment